java/runtime/src/main/java/org/ray/runtime/config/RayConfig.java

```diff
@@ -228,7 +229,7 @@ public String toString() {
   public static RayConfig create() {
     ConfigFactory.invalidateCaches();
     Config config = ConfigFactory.systemProperties()
-        .withFallback(ConfigFactory.load(CUSTOM_CONFIG_FILE))
+        .withFallback(ConfigFactory.load(StringUtil.isNullOrEmpty(System.getProperty("ray.config")) ? CUSTOM_CONFIG_FILE : System.getProperty("ray.config")))
```

Comment:
extract `System.getProperty("ray.config")` out and add logger.info, so help the user know which config file loaded.

Reply:
ok